src/compiler.adoc

```diff
@@ -846,8 +845,9 @@ Building ...
 Watching paths: /home/niwi/cljsbook/playground/leapyears/src
 ----
 
-You can observe that dommy dependency is successfully downloaded, and then the watch
-process is started.
+You can observe that the dommy dependency is successfully downloaded, and then the
+watch process is started.  In fact, Leiningen will ensure all dependencies are
+downloaded before performing any tasks.
```

Comment:
- article: (+the) dommy dependency
- elaborate: explain when lein downloads dependencies